Portfolio/index.html

```diff
@@ -6,8 +6,98 @@
     <link rel="stylesheet" href="index.css" />
```

Comment:
The inline styling should be avoided. It would be better to define the styling in a separate CSS file.